webapp/src/main/java/com/box/l10n/mojito/service/blobstorage/database/DatabaseBlobStorageCleanupJob.java

```diff
@@ -7,8 +7,10 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Profile;
 
 @DisallowConcurrentExecution
+@Profile("!disablescheduling")
```

Comment:
just wondering why it has the "!", other than that it looks good to me

Reply:
it runs "unless" the profile is set hence the negation